packages/webpack/lib/configs/develop.js

```diff
@@ -83,6 +83,7 @@ module.exports = function getConfig(config, name) {
     mode: 'development',
     context: config.rootDir,
     entry: [
+      require.resolve('../shims/event-source'),
```

Comment:
We can add the actual `eventsource` package here right away instead of adding another shim: it is being bundled anyway and will only attach itself to `window` if missing:

https://github.com/EventSource/eventsource/blob/82d38b0b0028ba92e861240eb6a943cbcafc8dff/lib/eventsource-polyfill.js#L5